### ISSUE TITLE
LC0005 - Wrong Warning for purchaseHeader

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#1168]
I now receive several warnings like the following:

> Wrong casing detected! Use purchaseHeader instead.

<img width="610" height="274" alt="Image" src="https://github.com/user-attachments/assets/81be7b1f-a3b3-4a11-adfb-da34f3175f61" />

Was anything changed here?

**Comment by pri-kise** [comment]
I could reproduce the issue.
```al
    procedure Dummy(var PurchaseHeader: Record "Purchase Header")
    begin
        MyProcedure1(PurchaseHeader);
        MyProcedure2(PurchaseHeader);
        MyProcedure3(PurchaseHeader);
    end;

    local procedure MyProcedure1(PurchaseHeader: Record "Purchase Header")
    begin
        if PurchaseHeader."No." = '' then
            exit;
    end;

    local procedure MyProcedure2(PurchaseHeader: Record "Purchase Header")
    begin
        if PurchaseHeader."No." = '' then
            exit;
    end;

    local procedure MyProcedure3(purchaseHeader: Record "Purchase Header")
    begin
        if PurchaseHeader."No." = '' then
            exit;
    end;
```



The wrong warning only occurs when the last procedure of a codeunit has a wrong casing.

I don't know why but then the warning is shown for any previous procedure.

**Comment by Arthurvdv** [comment]
Moved to ALCops
